app/src/main/java/com/keylesspalace/tusky/db/TimelineStatusEntity.kt

```diff
@@ -111,6 +111,8 @@ data class TimelineStatusWithAccount(
     val status: TimelineStatusEntity,
     @Embedded(prefix = "a_")
     val account: TimelineAccountEntity? = null, // null when placeholder
-    @Embedded(prefix = "rb_")
+    @Embedded(prefix = "rp_")
```

Comment:
Note: The source of this account data is a bit unexpected.

Most other information comes directly from the Api. This here is mixed-in only locally. This could be surprising?

Reply:
"rp_" is "reply"?

What's "bo_"? "boost"?

That's probably sufficiently confusing that it should just be "reply_" and "reblog_".

I prefer "reblog" to "boost" because we're pretty consistent about using the API terms for these things in the code, not the UI terms (e.g., it's always "status" not "post", etc).

Reply:
Ok, changed it to "author", "replied" and "reblogger" (all 3 TimelineStatusEntity).

(Generally these 2 queries and tedious to change and to understand.)